aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -341,6 +341,15 @@ Tensor new_empty(
   return at::empty(size, self.options().merge_in(options));
 }
 
+Tensor new_empty_strided(
+    const Tensor& self,
+    IntArrayRef size,
+    IntArrayRef strides,
```

Comment:
nit: size is singular but stride is plural in all the signature. Should we make them consistent?

Reply:
Changed to singular to match existing the signatures of existing strided functions (as_strided, empty_strided)